packages/mediapipe-core/third_party/mediapipe/tasks/c/components/processors/classifier_options.h

```diff
@@ -0,0 +1,61 @@
+/* Copyright 2023 The MediaPipe Authors.
```

Comment:
Indeed, third_party licensed code should be in a third_party folder.

https://opensource.google/documentation/reference/thirdparty/non-google3